src/css/imports/title.less

```diff
@@ -48,7 +47,5 @@
 
   .jw-title-secondary {
     display: none;
-    margin-top: 5px;
```

Comment:
Setting it here set's it once and prevents it from changing when the state changes which is a performance improvement.